qa/integration-tests/src/test/java/io/zeebe/broker/it/SecurityTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.grpc.netty.GrpcSslContexts;
+import io.netty.handler.ssl.SslContext;
+import io.zeebe.broker.Loggers;
+import io.zeebe.broker.it.clustering.ClusteringRule;
+import io.zeebe.client.ZeebeClientBuilder;
+import io.zeebe.client.api.response.Topology;
+import io.zeebe.gateway.impl.configuration.GatewayCfg;
+import javax.net.ssl.SSLException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.Timeout;
+
+public class SecurityTest {
+  public Timeout testTimeout = Timeout.seconds(120);
+
+  public ClusteringRule clusteringRule =
+      new ClusteringRule(
```

Comment:
Do we need to use 3 brokers? Only 1 makes the test a little bit faster, less setup.